Automattic/woocommerce-payments
# Add card reader print receipt branding logo

Fixes #3559

#### Changes proposed in this Pull Request

- Adding merchant branding logo on the top of the receipt.
- Adding a “Powered by WooCommerce” message on the bottom of the receipt.

<img width="329" alt="Screenshot 2022-02-11 at 18 03 16" src="https://user-images.githubusercontent.com/8667118/153626957-78cfb274-cafb-4ce4-8dc1-483d8a7d57bd.png">

#### Testing instructions
- Update the `_wcpay_feature_card_readers` option to `1`.
- Update the `card_present_eligible` to `true`.
- Update `isBrandingEnabled` feature flag to `true` in `client/card-readers/settings/index.js`.
- Navigate to `Payments` -> `Card Readers page` -> `Receipt details`.
- Scroll down to `Branding` and upload image file to `Logo` field.
- Save settings.
- Create an order and pay it with card reader following testing on [IPP Styled Receipts: Print receipt](https://github.com/Automattic/woocommerce-payments/pull/3433)
- Using Postman make a GET request to `/wp-json/wc/v3/payments/readers/receipts/<payment_id>`.
- Verify the Logo on top and “Powered by WooCommerce” message on bottom are displayed.
-------------------
